src/tools/ScenarioMeasurement/Startup/Startup.csproj

```diff
@@ -8,6 +8,13 @@
     <LangVersion>Preview</LangVersion>
   </PropertyGroup>
 
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+      <_Parameter1>Startup.Tests</_Parameter1>
+    </AssemblyAttribute>
```

Comment:
what's left that's internal that we need this for? Should we just make it public? What about using Moq or something similar?

Reply:
All the parser classes are internal. Sure we can make it public. My understanding of them being set "internal" originally was to minimize their scope and prevent external assemblies from using them?

Reply:
Eh, I don't think it's that principled. Ultimately there's no great reason for those types to be internal.